dungeon/assetlist.json

```diff
@@ -0,0 +1,26 @@
+{
+    "chain_id": "dungeon-1",
```

Comment:
the assetlist should have chain_name, not chain_id

Reply:
updated